pages/report-repair/[route].js

```diff
@@ -211,6 +211,8 @@ function ReportRepair() {
             {value: 'electricsExtractorCords', title: 'Electrics, including extractor fan and pull cords'},
             { value: 'windows', title: 'Damaged or stuck windows'},
             commonProblems.sink,
+            {value: 'electricsExtractorCords', title: 'Electrics, including extractor fan and pull cords'},
```

Comment:
This duplicates line [211](https://github.com/City-of-Lincoln-Council/housing-repairs-online-frontend/pull/55/files#diff-54de87070147bd6483cb396dde9b615c82d0684d7ef8dd228e327ea5b0d36ddcR211) and should be removed.
```suggestion
```